src/python/pants/pantsd/service/scheduler_service.py

```diff
@@ -189,12 +196,22 @@ def _process_event_queue(self):
 
         self._event_queue.task_done()
 
+    def _check_invalidation_watcher_liveness(self):
```

Comment:
```suggestion
    def _check_invalidation_watcher_liveness(self) -> None:
```

Why add this? Main reason is that MyPy will then check the body of the function. It also better signals intent to the reader.